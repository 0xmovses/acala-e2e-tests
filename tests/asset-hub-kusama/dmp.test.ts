import { buildTests } from './shared'
const tests = [
  {
    from: 'kusama',
    to: 'kusama-asset-hub',
    token: 'DOT',
  }
] as const

buildTests(tests)