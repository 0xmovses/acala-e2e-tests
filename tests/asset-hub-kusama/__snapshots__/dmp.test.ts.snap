// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`'kusama' -> 'kusama-asset-hub' xcm transfer 'DMP' > xcmPallet transfer > balance on fromChain 1`] = `
{
  "consumers": 0,
  "data": {
    "flags": "0x80000000000000000000000000000000",
    "free": "(rounded 9000000000000)",
    "frozen": 0,
    "reserved": 0,
  },
  "nonce": 1,
  "providers": 1,
  "sufficients": 0,
}
`;

exports[`'kusama' -> 'kusama-asset-hub' xcm transfer 'DMP' > xcmPallet transfer > balance on toChain 1`] = `
{
  "consumers": 0,
  "data": {
    "flags": 0,
    "free": 10000000000000,
    "frozen": 0,
    "reserved": 0,
  },
  "nonce": 0,
  "providers": 1,
  "sufficients": 0,
}
`;

exports[`'kusama' -> 'kusama-asset-hub' xcm transfer 'DMP' > xcmPallet transfer > tx events 1`] = `
[
  {
    "data": {
      "outcome": {
        "Complete": {
          "proofSize": "(rounded 6200)",
          "refTime": "(rounded 630000000)",
        },
      },
    },
    "method": "Attempted",
    "section": "xcmPallet",
  },
]
`;
